src/framework/analyses.ml

```diff
@@ -451,13 +451,18 @@ sig
 
   val special : (D.t, G.t, C.t, V.t) ctx -> lval option -> varinfo -> exp list -> D.t
   val enter   : (D.t, G.t, C.t, V.t) ctx -> lval option -> fundec -> exp list -> (D.t * D.t) list
-  val combine : (D.t, G.t, C.t, V.t) ctx -> lval option -> exp -> fundec -> exp list -> C.t option -> D.t -> Queries.ask -> D.t
+  val combine : (D.t, G.t, C.t, V.t) ctx -> ?longjmpthrough:bool -> lval option -> exp -> fundec -> exp list -> C.t option -> D.t -> Queries.ask -> D.t
```

Comment:
`longjmpthrough` (if really necessary at all) has no value in being an optional argument because every analysis must repeat the default value then. Worse yet, different analyses could even set the default differently.